projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -549,6 +550,24 @@ function wpcom_is_task_option_completed( $task ) {
 	return false;
 }
 
+/**
+ * Determine whether to show the design_edited task.
+ *
+ * @return bool True if the design_edited task should be shown.
+ */
+function wpcom_is_design_edited_visible() {
+	$task_statuses    = get_option( 'launchpad_checklist_tasks_statuses', array() );
```

Comment:
I think would be nice to have a function to get this option, since we are using it constantly.